security/oc_acl.c

```diff
@@ -365,8 +365,12 @@ oc_sec_check_acl(oc_method_t method, oc_resource_t *resource,
   /*  Retrieve requests to "/oic/res", "/oic/d" and "/oic/p" shall be
       granted.
   */
-  if (pstat->s == OC_DOS_RFOTM && method == OC_GET &&
-      ((oc_string_len(resource->uri) == 8 &&
+  if (pstat->s == OC_DOS_RFOTM && method == OC_GET && (
+#ifdef OC_WKCORE
+       (oc_string_len(resource->uri) == 17 &&
+        memcmp(oc_string(resource->uri), "/.well-known/core", 17) == 0) ||
+#endif
+       (oc_string_len(resource->uri) == 8 &&
```

Comment:
Would you mind updating the comment just above?

Reply:
done